src/Pages/seyongyun/Main/Components/comment/Comment.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import './Comment.scss';
+import '../../../../../Styles/seyongyun/common.scss';
+
+class Comment extends React.Component {
+  render() {
+    return (
+      <li key={this.props.indexContainer} className="replyElement">
+        <span>id </span>
+        <span>{this.props.textContainer}</span>
+      </li>
```

Comment:
```suggestion
  render() {
  const {indexContainer, textContainer} = this.props
    return (
      <li key={indexContainer} className="replyElement">
        <span>id </span>
        <span>{textContainer}</span>
      </li>
```

props를 넘겨줄때도 구조분해 할당을 하시면 조금 더 간편하게 작성하실 수 있을 것 같습니다 :)